guide/src/quickstart.md

```diff
@@ -4,38 +4,57 @@
 
 Install Creusot and why3 as described in the [README](https://github.com/creusot-rs/creusot).
 
-## Write specifications
+## Create a project
 
-Create a new project, and import creusot-contracts to start writing specifications:
+Create a new project with this command:
+
+```
+cargo creusot new project-name --creusot-contracts /PATH/TO/CREUSOT/creusot-contracts
```

Comment:
Why is a path necessary? Can't we use the version on crates.io?

Reply:
It's missing the new ghost primitives. Waiting for 0.4 :)